clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/ui/UTBotRequestProgressIndicator.kt

```diff
@@ -32,15 +33,21 @@ class UTBotRequestProgressIndicator(
     }
 
     fun stopShowingProgressInUI() {
-        finish()
-        super.stop()
+        // stop can be called only if progress was started (=running)
+        if (isRunning) {
+            finish()
+            super.stop()
```

Comment:
The call of `super` notation outside overwritten method looks suspicious.

Reply:
fixed: now I make sure the super.start is called in start(), and do not check for running status